tracing-appender/src/rolling.rs

```diff
@@ -37,6 +37,19 @@ impl io::Write for RollingFileAppender {
     }
 }
 
+/// A function which creates a hourly rolling file appender. This appender implements only the
+/// `Write` trait and can be used with `NonBlocking` to provide a `MakeWriter` to be used with a
+/// tracing subscriber.
+/// # Examples
+/// ``` rust,ignore
+/// let appender = rolling::hourly("/some/path", "rolling.log");
+/// let (non_blocking_appender, _guard) = non_blocking(appender);
```

Comment:
as i mentioned above, these will need to be referenced with their full paths:
```suggestion
/// let appender = tracing_appender::rolling::hourly("/some/path", "rolling.log");
/// let (non_blocking_appender, _guard) = tracing_appender::non_blocking(appender);
```